internal/agent/reset.go

```diff
@@ -70,6 +73,17 @@ func Reset() error {
 		args = append(args, "--reset-persistent")
 	}
 
+	cloudInit, ok := options["cc"]
```

Comment:
this is not going to work - config during reset is in the system, so to retrieve it you have to scan it in the system like https://github.com/kairos-io/kairos/blob/ca53fe6c13544610022e7259debfc7a5572c5e4d/internal/agent/notify.go#L15